### DISCUSSION TITLE
A guide for chromeOS

### DISCUSSION THREAD

**Discussion by Capedhunter** [IceHacks/SurvivCheatInjector#839] [Q&A]
so  i have been trying to i launch the appimage on  My chromeOS  which is 32-bit and i having issues launching it  can anyone here give a quick guide on how to run this on chromeOS? and is this only for 64-bit linux models

**Comment by IceHacks** [comment]
There is no official support for ChromeOS (however some can run AppImages). Yes, this app is 64-bit only atm.
